tests/testapp/test_helpers.py

```diff
@@ -15,35 +15,37 @@ def __new__(cls, id_):
         return super(WithId, cls).__new__(cls, id_, 'type name')
 
 
-def run_testcase_with_django_runner(
```

Comment:
@belugame fair enough, so let's leave it as is. 

How about the other two comments on this PR?